lib/WooCommerce/TransactionalEmails.php

```diff
@@ -0,0 +1,833 @@
+<?php
+
+namespace MailPoet\WooCommerce;
+
+use MailPoet\Config\Env;
+use MailPoet\Models\Newsletter;
+use MailPoet\Settings\SettingsController;
+use MailPoet\WP\Functions as WPFunctions;
+
+class TransactionalEmails {
+  const SETTING_EMAIL_ID = 'woocommerce.transactional_email_id';
+
+  /** @var WPFunctions */
+  private $wp;
+
+  /** @var SettingsController */
+  private $settings;
+
+  function __construct(WPFunctions $wp, SettingsController $settings) {
+    $this->wp = $wp;
+    $this->settings = $settings;
+  }
+
+  function init() {
+    $saved_email_id = (bool)$this->settings->get(self::SETTING_EMAIL_ID, false);
+    if (!$saved_email_id) {
+      $email = Newsletter::createOrUpdate([
```

Comment:
Shouldn't this be a newsletter template? That is what the specification says.

Also, why did you decide to keep the file here instead of having it somewhere in the `Config` where the rest of the templates live?